segmentation/utils/spatial_analysis_utils.py

```diff
@@ -87,22 +87,22 @@ def compute_close_cell_num(dist_mat, dist_lim, num, analysis_type,
     index [0, 1]).
 
     Args:
-        dist_mat: cells x cells matrix with the euclidian
+        dist_mat (np array): cells x cells matrix with the euclidian
             distance between centers of corresponding cells
-        dist_lim: threshold for spatial enrichment distance proximity
-        num: number of markers or cell phenotypes, based on analysis
-        analysis_type: type of analysis, either Cluster or Channel
-        fov_data: data for specific patient in expression matrix
-        fov_channel_data: data of only column markers for Channel Analysis
-        pheno_codes: list all the cell phenotypes in Cluster Analysis
-        thresh_vec: matrix of thresholds column for markers
+        dist_lim (int): threshold for spatial enrichment distance proximity
+        num (int): number of markers or cell phenotypes, based on analysis
+        analysis_type (string): type of analysis, either Cluster or Channel
+        current_fov_data (pandas df): data for specific patient in expression matrix
+        current_fov_channel_data (pandas df): data of only column markers for Channel Analysis
+        pheno_codes (pandas df): all the cell phenotypes in Cluster Analysis
```

Comment:
Make sure to update the variable names in all of these helper functions from the comments I left above